index.js

```diff
@@ -54,4 +59,37 @@ function updateMatchListElement() {
   matchListElement.appendChild(newMatchItem);
 }
 
+function addTeamsToTeamList({ teamA, teamB }) {
+  if (!teamList.find((element) => element.name === teamA)) {
+    teamList.push({ name: teamA, wins: 0 });
+  }
+
+  if (!teamList.find((element) => element.name === teamB)) {
```

Comment:
```suggestion
  if (!teamList.find((team) => team.name === teamB)) {
```
C'est bien d'etre le plus précis possible quand on sait ce qu'on récupère

Reply:
Okay